src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -131,77 +132,134 @@ function paginatePages(page, files, field, layout, ariaLabel, perPage) {
+function createHealthCareRegionListPages(page, drupalPagePath, files) {
   const relatedLinks = { fieldRelatedLinks: page.fieldRelatedLinks };
+  const sidebar = { facilitySidebar: page.facilitySidebar };
 
-  // Create the detail page for healthcare local facilities
   if (page.mainFacilities !== undefined || page.otherFacilities !== undefined) {
+    // Create the detail page for health care local facilities
     for (const facility of [
       ...page.mainFacilities.entities,
       ...page.otherFacilities.entities,
     ]) {
       if (facility.entityBundle === 'health_care_local_facility') {
-        const facilityCompiled = Object.assign(facility, relatedLinks);
-
         const pagePath = facilityLocationPath(
           drupalPagePath,
           facility.fieldFacilityLocatorApiId,
           facility.fieldNicknameForThisFacility,
         );
+
+        const facilityCompiled = Object.assign(facility, relatedLinks, sidebar);
+
         files[`drupal${pagePath}/index.html`] = createFileObj(
           facilityCompiled,
           'health_care_local_facility_page.drupal.liquid',
         );
       }
     }
+
+    // Create the top-level locations page for Health Care Regions
+    const locEntityUrl = createEntityUrlObj(drupalPagePath);
+    const locObj = Object.assign(
+      { mainFacilities: page.mainFacilities },
+      { otherFacilities: page.otherFacilities },
+      { fieldLocationsIntroBlurb: page.fieldLocationsIntroBlurb },
+      { facilitySidebar: sidebar },
+      { entityUrl: locEntityUrl },
+      { title: page.title },
+    );
+    const locPage = updateEntityUrlObj(locObj, drupalPagePath, 'Locations');
+    files[`drupal${drupalPagePath}/locations/index.html`] = createFileObj(
+      locPage,
+      'health_care_region_locations_page.drupal.liquid',
+    );
   }
 
-  files[`drupal${drupalPagePath}/health-services/index.html`] = createFileObj(
-    page,
-    'health_care_region_health_services_page.drupal.liquid',
-  );
+  // Create Health Services Page
+  if (
+    page.specialtyCareHealthServices !== undefined ||
+    page.primaryCareHealthServices !== undefined ||
+    page.mentalHealthServices !== undefined
+  ) {
+    const entityUrl = createEntityUrlObj(drupalPagePath);
+    const hsObj = Object.assign(
+      { specialtyCareHealthServices: page.specialtyCareHealthServices },
+      { primaryCareHealthServices: page.primaryCareHealthServices },
+      { mentalHealthServices: page.mentalHealthServices },
+      { fieldClinicalHealthServi: page.fieldClinicalHealthCareServi },
+      { facilitySidebar: sidebar },
+      { entityUrl },
+      { title: page.title },
+    );
+    const hsPage = updateEntityUrlObj(hsObj, drupalPagePath, 'Health Services');
+    files[`drupal${drupalPagePath}/health-services/index.html`] = createFileObj(
+      hsPage,
+      'health_care_region_health_services_page.drupal.liquid',
+    );
+  }
+
+  // Press Release listing page
+  if (page.allPressReleaseTeasers !== undefined) {
```

Comment:
I don't think we want this conditional. Listing pages should be created even if they're empty. Otherwise we'll have broken links on the region page and/or sidebar menu.